### PR TITLE
feat: add new loading design to the legacy pt

### PR DESCRIPTION
**Pull request by cisel**
# Description
Ticket:https://rvu.atlassian.net/secure/RapidBoard.jspa?rapidView=80&projectKey=MONEY&modal=detail&selectedIssue=MONEY-146&quickFilter=147

**Wireframe**
Aug-11-2021 19-20-11
Aug-11-2021 19-27-12

**Other Examples**
To show they still work fine
Screenshot 2021-08-05 at 18 28 08
Screenshot 2021-08-05 at 18 28 21
Screenshot 2021-08-05 at 18 28 43
Screenshot 2021-08-05 at 18 28 58

# Checklist

Pull request contains:

- [ ] A new component
- [ ] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [ ] Includes theme changes for both Uswitch and money
- [ ] Work has been tested in multiple browsers
- [ ] PR description includes description of change
- [ ] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.

### CODE REVIEW

**Review by samhstn — COMMENTED** [review]
Looks nice, your build is failing though 🤔

File: src/compounds/legacy-product-table/src/index.tsx [hunk 1]
```diff
@@ -9,6 +9,128 @@ import CirclePercentageBar from '@uswitch/trustyle.circular-percentage-bar'
...
+    representativeExample: object
+  } = {
+    header: {
+      width: ['320px', '425px'],
+      height: '14px'
+    },
+    additionalInfo: {
+      height: '10px',
+      width: '100%',
+      maxWidth: ['326px', '653px'],
```
Comment (jonathan-fielding) [inline src/compounds/legacy-product-table/src/index.tsx:63]: these numbers feel a bit magic, where do they come from, can we get anything from the theme?

File: src/compounds/legacy-product-table/src/index.tsx [hunk 2]
```diff
@@ -9,6 +9,128 @@ import CirclePercentageBar from '@uswitch/trustyle.circular-percentage-bar'
...
+      marginLeft: ['0', 'auto'],
+      marginRight: 'auto',
+      marginTop: '10px'
+    },
+    representativeExample: {
+      height: '11px',
+      width: '100%',
+      maxWidth: ['261px', '785px'],
```
Comment (jonathan-fielding) [inline src/compounds/legacy-product-table/src/index.tsx:71]: these numbers feel a bit magic, where do they come from, can we get anything from the theme?

File: src/compounds/legacy-product-table/src/index.tsx [hunk 3]
```diff
@@ -9,6 +9,128 @@ import CirclePercentageBar from '@uswitch/trustyle.circular-percentage-bar'
...
+    }
+  }
+
+  const defaultStyles: {
+    header: object
+    additionalInfo: object
+    representativeExample: object
+  } = {
+    header: {
+      width: ['320px', '425px'],
```
Comment (jonathan-fielding) [inline src/compounds/legacy-product-table/src/index.tsx:57]: these numbers feel a bit magic, where do they come from, can we get anything from the theme?

File: src/compounds/legacy-product-table/src/index.tsx [hunk 4]
```diff
@@ -188,9 +310,20 @@ const BaseCell: React.FC<React.HTMLAttributes<any>> = ({
...
+  children: React.ReactNode
+  isWireFrame?: boolean
+}
+
+export const ImageCell: React.FC<ImageCellProps> = ({
+  children,
+  isWireFrame = false
 }) => {
+  const wireFrameStyles = {
+    width: '118px',
```
Comment (jonathan-fielding) [inline src/compounds/legacy-product-table/src/index.tsx:323]: these numbers feel a bit magic, where do they come from, can we get anything from the theme?

File: src/compounds/legacy-product-table/src/index.tsx [hunk 5]
```diff
@@ -9,6 +9,128 @@ import CirclePercentageBar from '@uswitch/trustyle.circular-percentage-bar'
...
+}
+
+const WireFrame: React.FC<WireFrameProps> = ({ type }) => {
+  const containerStyles: {
+    header: object
+    additionalInfo: object
+    representativeExample: object
+  } = {
+    header: {
+      paddingLeft: ['15px', '160px'],
```
Comment (jonathan-fielding) [inline src/compounds/legacy-product-table/src/index.tsx:36]: these numbers feel a bit magic, where do they come from, can we get anything from the theme?

File: src/compounds/legacy-product-table/src/index.tsx [hunk 6]
```diff
@@ -217,51 +351,91 @@ interface DataCellProps extends React.HTMLAttributes<HTMLDivElement> {
...
 }) => {
+  const wireFrameLabelStyles = {
+    width: '100%',
+    maxWidth: ['134px', '84px'],
+    height: '10px'
+  }
+
+  const wireFrameDescriptionStyles = {
+    width: '100%',
+    maxWidth: ['202px', '126px'],
```
Comment (jonathan-fielding) [inline src/compounds/legacy-product-table/src/index.tsx:373]: these numbers feel a bit magic, where do they come from, can we get anything from the theme?

File: src/compounds/legacy-product-table/src/index.tsx [hunk 7]
```diff
@@ -217,51 +351,91 @@ interface DataCellProps extends React.HTMLAttributes<HTMLDivElement> {
...
   backgroundColor,
   borderBottomColor,
   color,
-  label
+  label,
+  isWireFrame = false
 }) => {
+  const wireFrameLabelStyles = {
+    width: '100%',
+    maxWidth: ['134px', '84px'],
```
Comment (jonathan-fielding) [inline src/compounds/legacy-product-table/src/index.tsx:367]: these numbers feel a bit magic, where do they come from, can we get anything from the theme?

Comment (cisel) [reply src/compounds/legacy-product-table/src/index.tsx]: Turned them into percentages according to the design

File: src/compounds/legacy-product-table/src/index.tsx (hunk 6 above)
Comment (cisel) [reply src/compounds/legacy-product-table/src/index.tsx]: Turned them into percentages according to the design

File: src/compounds/legacy-product-table/src/index.tsx (hunk 5 above)
Comment (cisel) [reply src/compounds/legacy-product-table/src/index.tsx:37]: Turned them into percentages according to the design

File: src/compounds/legacy-product-table/src/index.tsx (hunk 4 above)
Comment (cisel) [reply src/compounds/legacy-product-table/src/index.tsx]: Turned them into percentages according to the design

File: src/compounds/legacy-product-table/src/index.tsx (hunk 3 above)
Comment (cisel) [reply src/compounds/legacy-product-table/src/index.tsx]: Turned them into percentages according to the design

File: src/compounds/legacy-product-table/src/index.tsx (hunk 2 above)
Comment (cisel) [reply src/compounds/legacy-product-table/src/index.tsx]: Turned them into percentages according to the design

File: src/compounds/legacy-product-table/src/index.tsx (hunk 1 above)
Comment (cisel) [reply src/compounds/legacy-product-table/src/index.tsx]: Turned them into percentages according to the design

File: src/compounds/legacy-product-table/src/index.tsx (hunk 5 above)
Comment (cisel) [reply src/compounds/legacy-product-table/src/index.tsx:37]: I got this from the current styling of the header on legacy product table

**Review by jonathan-fielding — DISMISSED** [review]

**Review by samhstn — APPROVED** [review]

**Pull request merged**